src/docs/windows/pwa-api-documentation/pwa-api-documentation.md

```diff
@@ -0,0 +1,312 @@
+  </p>
+</div>
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    Check out <a href="https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share" target="_blank" rel="noopener">this tutorial</a> to learn how to use the Web Share API and enable this functionality in your PWA
+  </p>
+</div>
+
+Your PWA can also register as a Share Target, enabling your PWA to have content shared directly to it.
+
+<div class="how-to">
+  <strong>How To: Share Target</strong>
+
+  <p>
+    Check out <a href="https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/webappmanifests#identify-your-app-as-a-share-target" target="_blank" rel="noopener">this tutorial</a> to learn how to use the Share Target API and enable this functionality in your PWA.
+  </p>
+</div>
+
+Finally, your PWA can use the early File Handlers API to register as a File Handler for certain file types. This enables you to open a file directly into a PWA, just as you can open images directly into the Photos app from the File Explorer:
+
+<div class="how-to">
+  <strong>How To: File Handling</strong>
+
+  <p>
+    Check out <a href="https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/experimental-features/#file-handling" target="_blank" rel="noopener">this documentation</a> to learn more!
+  </p>
+</div>
+
+## Shortcuts
+
+Using the Web Shortcuts API, PWAs can tie into the shortcuts UI in Windows, enabling users to jump to specific pages or actions in your PWA! This can be useful for
+
+- Mail Applications for example, as you can let users jump straight to writing a new email from a right click on your App Icon in the start menu and taskbar!
+- Social Media Apps such as Twitter, which enables users to jump straight into creating a tweet, the notifications screen, reading their DMs and more, all from a right click of the Twitter icon.
+
+<img src="/docs/windows/pwa-api-documentation/shortcuts.png" width="350px" alt="A screenshot of what the shorcuts UI looks like on Windows" /> 
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    Check out <a href="https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/webappmanifests#use-shortcuts-to-provide-quick-access-to-features" target="_blank" rel="noopener">this tutorial</a> to learn how to add shortcuts to your PWA.
+  </p>
+</div>
+
+## The Clipboard
+
+Want to be able to programmatically copy or paste content to and from your PWA? While this was possible before with an older clipboard API built into browsers, the experience was not great. With the new Async Clipboard API it is much easier to programmatically copy and paste everything from text to files. This also works seamlessly with the Cloud Clipboard in Windows!
+
+<div class="demo-block">
+  <strong>Demo</strong>
+
+  <p>
+    Check out a demo <a href="https://wiggly-impossible-crayfish.glitch.me/clipboard.html" target="_blank" rel="noopener">here</a> of the Clipboard API!
```

Comment:
In the clipboard demo, can you show a "copied  to the clipboard" text when something is copied? When I tried the demo, it wasn't clear if it was working when clicking the copy buttons.

Reply:
Added